team replay q2 2025 (#11083)
PostHog/posthog.com: contents/teams/replay/objectives.mdx

# 🚀 Goal 1: Get Alex 2 rocketing 2

it's great to have Alex 2 on the team... let's make it a success

# 👩‍🔬 Goal 2: Find the needle in the haystack

* Understand the needles that people are looking for using user interviews, data analysis and more 
* Watch recordings from where you are (e.g. in funnels or experiments) 
* some people are scared of filtering, some people have no patience for it - give them ways to avoid it

> * show me, don’t make me figure it out - make it super easy to see what filters have results to watch

Author: ```suggestion
* show me, don’t make me figure it out - make it super easy to see what filters have results to watch

see more detail in our mega issue https://github.com/PostHog/posthog/issues/30460
```

> # 🚀 Goal 1: Get Alex 2 rocketing 2

Reviewer: The replay team can only hire an Alex for the indefinite future 😆💖

it's great to have Alex 2 on the team... let's make it a success

# 👩‍🔬 Goal 2: Find the needle in the haystack

> 

Reviewer 2: ```suggestion
Many of our customers have lots of recordings, but struggle to find the "useful" ones. We want to test and iterate  on a bunch of different ideas to get customers to value faster.

```

* Understand the needles that people are looking for using user interviews, data analysis and more 

> * Watch recordings from where you are (e.g. in funnels or experiments) 

Reviewer 2: ```suggestion
* Better options for users to watch recordings from where they are (e.g. in funnels or experiments) 
```

> * some people are scared of filtering, some people have no patience for it - give them ways to avoid it

Reviewer 2: ```suggestion
* Some people are scared of filtering, some people have no patience for it - give them ways to avoid it
```

> * show me, don’t make me figure it out - make it super easy to see what filters have results to watch

Reviewer 2: ```suggestion
* Make it super easy to see what filters have results to watch - "show me, don’t make me figure it out"
```

# 🚀 Goal 1: Get Alex 2 rocketing 2

it's great to have Alex 2 on the team... let's make it a success

# 👩‍🔬 Goal 2: Find the needle in the haystack

> Many of our customers have lots of recordings, but struggle to find the "useful" ones. We want to test and iterate  on a bunch of different ideas to get customers to value faster.

Author: ```suggestion

Many of our customers have lots of recordings, but struggle to find the "useful" ones. We want to test and iterate  on a bunch of different ideas to get customers to value faster.
```
